### PR TITLE
Add gqrx.rb cask

### PR DESCRIPTION
**Pull request by yuvadm**
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

**Comment by yuvadm** [comment]
This cask was removed in #9021, but can be restored now that releases are properly done on github.

File: Casks/gqrx.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cask 'gqrx' do
+  version '2.5.3'
+  sha256 'b5053b9a86c13d856137143a8215f48a85d3e57b4d867c317a666b1bc0e97d10'
+
+  url "https://github.com/csete/gqrx/releases/download/v#{version}/Gqrx-#{version}-2.dmg"
```
Comment (adidalal) [inline Casks/gqrx.rb]: Needs a `# github.com/csete/gqrx was verified as official when first introduced to the cask` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/4ad02c9f087deb9e553ab0129c079d73c474add8/Casks/0ad.rb#L5); [reasoning](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md#when-url-and-homepage-hostnames-differ-add-a-comment)).

Comment (yuvadm) [inline Casks/gqrx.rb]: @adidalal I'll add the comment, no problem. As for the `2` part, I believe it's a packaging revision by the developer. Is there an example of how to use version helpers in this case?

Comment (adidalal) [inline Casks/gqrx.rb]: Should be OK as is actually, let's see if/how that changes with new releases - we can always change it later.

Just add the url comment and comment on the PR so I get a ping.

**Comment by adidalal** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 1f3dee477f5d877d0505135733ff4a9db4b88ce3. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**